src/components/ifxToggle/ifxToggle.tsx

```diff
@@ -0,0 +1,28 @@
+// pill-toggle.tsx
+import { Component, Prop, h, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-toggle',
+  styleUrl: 'ifxToggle.scss',
+  shadow: true,
+})
+export class ifxToggle {
+  @Prop({ mutable: true }) checked: boolean = false;
+
+  @Event({ eventName: 'valueChanged' }) valueChanged: EventEmitter<boolean>;
+
+  toggle() {
+    this.checked = !this.checked;
+    this.valueChanged.emit(this.checked);
+
+    console.log(this.checked);
+  }
+
+  render() {
+    return (
+      <div class={`pill-container ${this.checked ? 'checked' : ''}`} onClick={() => this.toggle()}>
```

Comment:
Why are the classes prefixed with `pill-`and not `toggle-`?

Reply:
copy & paste ;)